examples/nextjs/pages/locations/ConfigScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useCallback, useState, useEffect } from 'react';
```

Comment:
Why are all components under `/pages/locations`? wouldn't that mean theoretically there is now a route to `/pages/locations/ConfigScreen` ?

Reply:
I don't understand? This is just structured the same as the React template.

Reply:
In the other templates we only have `src/locations/<location>.tsx`, but here it is `src/pages/locations/<location>.tsx`, I just wonder why we need that `pages` folder as we don't have routing

Reply:
tbh, I have no experience about Next.js and just put the files into the folder that was created via create-next

Reply:
The idea is that if you put things in the `pages` folder  automatically creates the route like that.
https://nextjs.org/docs/basic-features/pages

That means we technically now have routes like `/pages/locations/ConfigScreen` available which we don't need. 
So having the same structure as the other js/ts templates should be working here as well

Reply:
Okay, gonna move it. Thanks

Reply:
done